CODEOWNERS

```diff
@@ -931,6 +931,8 @@ build.json @home-assistant/supervisor
 /homeassistant/components/octoprint/ @rfleming71
 /tests/components/octoprint/ @rfleming71
 /homeassistant/components/ohmconnect/ @robbiet480
+/homeassistant/components/ollama_conversation/ @synesthesiam
```

Comment:
let's rename the integration to `ollama`. I don't know why I added `conversation` to `openai` but let's not repeat that mistake 👍